dragonchain/lib/dto/schema.py

```diff
@@ -671,3 +671,62 @@ def get_transaction_queue_task_schema(dict_payload: bool = False) -> Dict[str, A
     "required": ["to", "value"],
     "additionalProperties": False,
 }
+
+# BINANCE INTERCHAIN #
+
+create_binance_interchain_schema_v1 = {
+    "type": "object",
+    "properties": {
+        "version": {"type": "string", "enum": ["1"]},
+        "name": {"type": "string"},
+        "testnet": {"type": "boolean"},
+        "rpc_address": {"type": "string"},
+        "api_address": {"type": "string"},
+        "private_key": {"type": "string"},
+    },
+    "additionalProperties": False,
+    "required": ["name", "version"],
+}
+
+# Same as create without required name field
+update_binance_interchain_schema_v1 = {
+    "type": "object",
+    "properties": {
+        "version": {"type": "string", "enum": ["1"]},
+        "name": {"type": "string"},
+        "testnet": {"type": "boolean"},
+        "rpc_address": {"type": "string"},
+        "api_address": {"type": "string"},
+        "private_key": {"type": "string"},
+    },
+    "additionalProperties": False,
+    "required": ["version"],
+}
+
+
+bnb_transaction_schema_v1 = {
+    "type": "object",
+    "properties": {
+        "wallet": {"type": "object"},
```

Comment:
private key is associated with the network, so this shouldn't need to be submitted when creating a transaction for a network

Reply:
Removed -- also added in missing memo field, fixed required..